index.html

```diff
@@ -888,44 +888,44 @@ <h3 id="toc_5">Scenario 4+: Masks For All, Summer, Circuit Breakers</h3>
 <div id="footer">
 
 	<div>
-		
-		Help this post get R > 1: <span class='shareables in_footer' style='position: relative; top: 17px; right: -6px;'></span>
+
+		Aider ce guide à atteindre R > 1 : <span class='shareables in_footer' style='position: relative; top: 17px; right: -6px;'></span>
 		<br><br>
-		This guide is
+		Ce guide est dans le:
 		<br>
 		<img src='sharing/cc0.png' style="margin-right: 10px; margin-top: 10px; width: 120px;"/>
 		<b style='font-size: 100px;position: relative;top: -24px;'>
-			PUBLIC DOMAIN
+			DOMAINE PUBLIC
 		</b>
-		That means you <i>already</i> have permission to re-use &amp; remix
-		any of the art/code/words on this page – on blogs, news sites, classrooms, anywhere!
+		Cela signifie que vous avez <i>déjà</i> la permission de réutiliser &amp; ré-arranger
+		n'importe quel graphique/code/texte sur cette page – sur des blogs, des sites d'actualité, des salles de classe, n'importe où !
 		<a href='https://github.com/ncase/covid-19'>
-			(Swipe our code on Github)
+			(Venez voir le code sur Github)
 		</a>
 		<br>
 		<b>
-			Just remember to credit
+			N'oubliez simplement pas de créditer :
 			<a href='https://scholar.google.com/citations?user=_wHMGkUAAAAJ&hl=en'>Marcel Salathé</a>
 			&
 			<a href='https://ncase.me/'>Nicky Case</a>
-			(May 2020)
+			(Mai 2020)
 		</b>
-		
-		
+
+
 		<br><br>
 
-		This free, open-source guide was made possible thanks to
-		Nicky's supporters on Patreon.
-		If (seriously, ONLY IF) you have some disposible income in these hard times, you can
-		<a href='https://www.patreon.com/ncase'>throw coins at Nicky!</a>
-		If not, you can share this guide or
-		<a href='https://ncase.me/'>see their other educational art/interactives</a>.
-		(excellent for homeshooling if, say, schools are currently closed
-		due to a pandemic)
+		Ce guide, libre et gratuit, a été rendu possible grâce aux
+		supporteurs de Nicky sur Patreon.
+		Si (et, vraiment, seulement SI) vous avez quelques revenus disponibles durant ces temps difficiles, vous pouvez :
+		<a href='https://www.patreon.com/ncase'>Jeter quelques pièces à Nicky !</a>
+		Sinon, vous pouvez partager ce guide, ou
+		<a href='https://ncase.me/'>voir ses autres oeuvres artistiques/interactives pédagogiques </a>.
+		(excellent pour l'école à la maison, dans l'éventualité où les écoles seraient fermée
+		à cause d'une pandémie)
 
 		<br><br>
 
-		Here's some of the generous Patrons who made this possible: 💖
+		Voilà quelques-uns des généreux donateurs qui ont rendu cela possible : 💖
```

Comment:
je  suis chaud pour enlever les donateurs perso

Reply:
Tu enlèverais toute la liste ? ou tu parles du mot dans la phrase ?

Reply:
toute la liste

Reply:
Décider unilatéralement d'enlever cette liste risque d'être vu comme "harsh" par Nicky, suivant le rapport entretenu vis-à-vis de ses supports. Je pense que c'est à Nicky de donner une guideline pour ça... qu'en penses-tu ?

Reply:
Juste au dessus c'est écrit:  
That means you already have permission to re-use & remix any of the art/code/words on this page – on blogs, news sites, classrooms, anywhere! (Swipe our code on Github)
Just remember to credit Marcel Salathé & Nicky Case (May 2020)

et c'est en creative commons 0 donc nicky est bien conscient qu'on peut faire ce qu'on veut avec ce qu'il a fait. imho ça prend de la place, personne ne vas regarder, et les supporters n'ont pas forcément envie de se retrouver dans des traductions non officielles.

Reply:
je serais plutôt de l'avis de @e-guenat...au final c'est Nicky qui va accepter la Pull Request et la publier sur son site :)

Reply:
Certes c'est en cc0, mais on étiquette notre travail comme une traduction d'un autre contenu, ce qui suggère (selon moi hein) que les modifications devraient se réduire à la transformation du texte d'une langue à l'autre et pas beaucoup plus...

Reply:
traduttore traditore mais ok on laisse